database/factories/UserFactory.php

```diff
@@ -13,6 +13,18 @@
     ];
 });
 
+$factory->state(App\User::class, 'enabledNotifications', function (Faker $faker) {
```

Comment:
I believe there's a way to make a state extend another, so you can just get the result of the first state and just modify it. If you don't know about this, let me know and I'll look it up :)

Reply:
I tried to look it up, but with no success yet. Do you find it?

Reply:
<img width="828" alt="image" src="https://user-images.githubusercontent.com/151829/48077801-28d2c800-e1b6-11e8-9155-3adc76bf8fb6.png">

This is from my book. Hope it's right :)

Reply:
Hehe nice when you can look up things in your own book 😅Thanks, will try it out.

Reply:
It didn't work for me with the array as the third argument, but with a function it works. It seems the `define` sets the default values and in the `state` we can now override it.  This way the factories get a little bit cleaner 👍